test/example.niceorg.html

```diff
@@ -35,6 +35,12 @@
 
 <body>
 
+
+	<div id="main-content">
```

Comment:
If we're using this `main-content` then we should also set `data-skip-link-id="main-content"` on the tophat script, otherwise the `main-content` div is redundant

Reply:
This has been done to demonstrate that the default data-skip-link-id will take your focus to the end of tophat which we expect to be immediately proceeded by some kind of main content. This div can actually be removed, its just for demonstration.